src/App.js

```diff
@@ -4,21 +4,19 @@ import Board from "./components/Board";
 
 const App = () => {
   const [boardName, setBoardName] = useState("shonds-dubs");
-  {
```

Comment:
removed this extra {} from this section to get rid of the all the line errors on the linter